README.md

```diff
@@ -1,2 +1,88 @@
 # mtrho-operator
-Migration Toolkit for Kubernetes Operator
+Migration Toolkit for Red Hat OpenShift operator.
+
+#### Prerequisite
+
+<b>Note</b>: 
+- mtRHO is compatible with OpenShift 4.10/4.10+ versions.
+- Go 1.18 version needed to build operator
+
+
+
+1. Install pipeline operator from Operator-Hub
+2. Make sure `CusterTask`, `Pipeline` CRDs are available after the installation of Pipeline operator. It might take a minute or two before the CRDs appear.
```

Comment:
```suggestion
2. Make sure `ClusterTask`, `Pipeline` CRDs are available after the installation of Pipeline operator. It might take a minute or two before the CRDs appear.
```